appserver/monitoring-console/core/src/main/java/fish/payara/monitoring/store/InMemoryMonitoringDataRepository.java

```diff
@@ -426,7 +433,7 @@ private static boolean isRelevantSet(SeriesDataset set, Set<String> instanceFilt
         String[] series;
         long[] values;
         // annotations
-        List<SeriesAnnotation> annotations;
+        ArrayList<SeriesAnnotation> annotations;
```

Comment:
NB. because of serialisation the type here must be a type known to be `Serializable`